CSharpException/CSharpException/Triangle.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace CSharpException
+{
+    public class Triangle
+    {
+        private double sideA;
+        private double sideB;
+        private double sideC;
+
+        public double SideA
+        {
+            get { return sideA; }
+            set { sideA = value; }
+        }
+
+        public double SideB
+        {
+            get { return sideB; }
+            set { sideB = value; }
+        }
+
+        public double SideC
+        {
+            get { return sideC; }
+            set { sideC = value; }
+        }
+
+
+        public void ValidateTriangle()
+        {
+
+            if (SideA >= SideB + SideC || SideB >= SideA + SideC || SideC >= SideA + SideB)
+            {
+                throw new TriangleException("The sum of any two sides of a triangle must be greater than the third side.");
+            }
+            else
+            {
+                Console.WriteLine($"Sides of triangle: {SideA}, {SideB}, {SideC}");
+            }
+        }
+
+        public void ReadSides()
+        {
+            Console.WriteLine("Enter Side A length: ");
+            double inputA;
+            while (!double.TryParse(Console.ReadLine(), out inputA) || inputA <= 0)
```

Comment:
Better to have another way to verify numbers. Introduce a new bool variable(isNumberNull) and one loop (while(!isNumberNull)) for all numbers. And if the user enter 0 makes this variable true and add "continue;" in if construction to return to condition in while loop to enter number again.